docs/basic-docs/atomicdex-api-20/enable_tendermint_token.md

```diff
@@ -0,0 +1,82 @@
+# enable\_tendermint\_token
+
+The `enable_tendermint_token` method allows you to activate additional Tendermint assets. Before using this method, you first need to use the [enable_tendermint_with_assets](enable_tendermint_with_assets.html) method.
```

Comment:
looks like `enable_tendermint_with_assets` link shows `404 error` should  there be a `enable_tendermint_with_assets.md` instead of `enable_tendermint_with_assets.html` ?

Reply:
once deployed this should work (right @gcharang ?)

Reply:
@Canialon I just tested it and it works. can you please try it again? might be the OS it is being built on could be affecting the link behavior. we will be deploying it in linux and I tested it locally in my linux machine